sematic/resolvers/resource_requirements.py

```diff
@@ -69,15 +214,42 @@ class KubernetesResourceRequirements:
     secret_mounts:
         Requests to take the contents of Kubernetes secrets and expose them as
         environment variables or files on disk when running in the cloud.
+    tolerations:
+        If your Kubernetes configuration uses node taints to control which workloads
+        get scheduled on which nodes, this enables control over how your workload
+        interacts with these node taints. More information can be found here:
+        https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/
     """
 
     node_selector: Dict[str, str] = field(default_factory=dict)
     requests: Dict[str, str] = field(default_factory=dict)
     secret_mounts: KubernetesSecretMount = field(default_factory=KubernetesSecretMount)
+    tolerations: List[KubernetesToleration] = field(default_factory=list)
+
+    def assert_valid(self):
```

Comment:
Same here, should we do this on istantiation?

Reply:
Moved to `__post_init__`